src/Client/components/ques/written-ques.tsx

```
@@ -1,143 +1,200 @@
-              isShortAnswer: !isChecked,
-              longAnswerText: isChecked ? '' : undefined,
-              shortAnswerText: undefined,
-            }
-          : question
-      );
-    });
-  };
-
-  const totalPages = useMemo(() => Math.ceil(writtenQuestions.length / questionsPerPage), [writtenQuestions, questionsPerPage]);
-  const currentQuestions = useMemo(() => writtenQuestions.slice(currentPage * questionsPerPage, (currentPage + 1) * questionsPerPage), [writtenQuestions, currentPage, questionsPerPage]);
+        question.id === questionId ? { ...question, points: parseInt(points.toString()) } : question
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)